src/GraphQL.Conventions/GraphQL.Conventions.csproj

```diff
@@ -29,9 +29,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="5.3.0" />
-    <PackageReference Include="GraphQL.DataLoader" Version="5.3.0" />
-    <PackageReference Include="GraphQL.NewtonsoftJson" Version="5.3.0" />
+    <PackageReference Include="GraphQL" Version="7.0.0" />
+    <PackageReference Include="GraphQL.DataLoader" Version="7.0.0" />
+    <PackageReference Include="GraphQL.NewtonsoftJson" Version="7.0.0" />
```

Comment:
It would be better to switch to GraphQL.SystemTextJson.

Reply:
can we do that as a diff PR?

Reply:
Perhaps... but I would not switch it without the option to switch back.

Reply:
Yes it certainly should be a separate PR.  GraphQL.NewtonsoftJson is still supported.